File: sarcasm.js
function convertChar(c) {
    if (/[a-zA-Z]/.test(c)) {
        return Math.random() < 0.5 ? c.toUpperCase() : c.toLowerCase();
    }
    return c;
}

function convertString(input) {
    let output = '';
    for (let i = 0; i < input.length; i++) {
        output += convertChar(input[i]);
    }
    return output;
}

const input = prompt("Enter a string:");
const convertedOutput = convertString(input);
console.log("Converted output:", convertedOutput);

File: sarcasm.py
import random
import string

def convert_char(c):
    if c.isalpha():
        if random.choice([True, False]):
            return c.upper()
        else:
            return c.lower()
    return c

def convert_string(input_string):
    return ''.join(convert_char(c) for c in input_string)

input_string = input("Enter a string: ")
converted_output = convert_string(input_string)
print("Converted output:", converted_output)
